### PR TITLE
Np 5007 reorganize packages and classes

### PR DESCRIPTION
**Pull request by torarnet**

### CODE REVIEW

File: cristin-commons/src/main/java/no/unit/nva/cristin/common/client/CristinQueryApiClient.java [hunk 1]
```diff
@@ -1,9 +1,10 @@
 package no.unit.nva.cristin.common.client;
 
+import no.unit.nva.client.GenericApiClient;
 import no.unit.nva.cristin.model.SearchResponse;
 import nva.commons.apigateway.exceptions.ApiGatewayException;
 
-public interface QueryApiClient<T, R> extends GenericApiClient {
+public interface CristinQueryApiClient<T, R> extends GenericApiClient {
```
Comment (torarnet) [inline cristin-commons/src/main/java/no/unit/nva/cristin/common/client/CristinQueryApiClient.java:7]: was more tightly coupled to cristin than i thought. gave it a name which reflected that

File: cristin-project/src/main/java/no/unit/nva/cristin/projects/model/nva/ContributorRoleMapping.java [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package no.unit.nva.utils;
+package no.unit.nva.cristin.projects.model.nva;
```
Comment (torarnet) [inline cristin-project/src/main/java/no/unit/nva/cristin/projects/model/nva/ContributorRoleMapping.java:1]: this was only used in project module. no need to have it in commons module

**Comment by torarnet** [comment]
Mostly just movement between packages.

**Review by brinxmat — APPROVED** [review]
Good job on this; clear improvements and helpful notes!

**Pull request merged**